src/backend/keys/cswordversekey.h

```diff
@@ -159,6 +159,10 @@ class CSwordVerseKey final : public CSwordKey {
         long index() const { return m_key.getIndex(); }
         void setIndex(long v) { m_key.setIndex(v); }
         void positionToTop() { m_key.setPosition(sword::TOP); }
+        QString versification() { return m_key.getVersificationSystem(); }
```

Comment:
Can this getter be marked `const`?

Reply:
Yes it is somewhat important. I wonder i m able to write in C after 5 years at all, so your notes are very helpful.